CHANGELOG.md

```diff
@@ -57,9 +57,13 @@
 - Remove the requirement for INVALID_STATE for the cxx::expected [\#987](https://github.com/eclipse-iceoryx/iceoryx/issues/987)
 - Add unique test identifers [\#988](https://github.com/eclipse-iceoryx/iceoryx/issues/988)
 - Remove `ApplicationPort` and `ApplicationPortData` classes [\#415](https://github.com/eclipse-iceoryx/iceoryx/issues/415)
+- Updating Codecov API and enforce CMake version 3.16 for building iceoryx [\#774](https://github.com/eclipse-iceoryx/iceoryx/issues/774) and [\#1031](https://github.com/eclipse-iceoryx/iceoryx/issues/1031)
 
 **API Breaking Changes:**
 
+The CMake files in iceoryx expect to have CMake version 3.16 or greater installed, otherwise the build fails.
+(Hint: Ubuntu 18 users can install `cmake-mozilla` from the universe PPA)
```

Comment:
It is not a ppa but a repo provided by canonical. It's just off by default

Reply:
fixed it